InfiniteScrollView.js

```diff
@@ -116,18 +117,23 @@ class InfiniteScrollView extends React.Component {
   }
 
   _distanceFromEnd(event): number {
-    let {
+    const {
```

Comment:
These work as constants in javascript, no? They are never modified and changed so constant is a more suitable fit